config/set/php80.php

```diff
@@ -89,20 +85,6 @@
             'pg_result' => 'pg_fetch_result',
             'pg_setclientencoding' => 'pg_set_client_encoding',
         ]);
-    $rectorConfig->rule(OptionalParametersAfterRequiredRector::class);
-
-    $rectorConfig
-        ->ruleWithConfiguration(FunctionArgumentDefaultValueReplacerRector::class, [
-            new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, 'gte', 'ge'),
-            new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, 'lte', 'le'),
-            new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, '', '!='),
-            new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, '!', '!='),
-            new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, 'g', 'gt'),
-            new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, 'l', 'lt'),
-            new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, 'gte', 'ge'),
-            new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, 'lte', 'le'),
-            new ReplaceFuncCallArgumentDefaultValue('version_compare', 2, 'n', 'ne'),
-        ]);
```

Comment:
this is on php 8.0, which version_compare doesn't support `gte` and friend, see https://3v4l.org/OJbSg#v8.0.0 vs https://3v4l.org/Jufeb#v8.0.0